test/ExportQueryExecutionTreesTest.cpp

```diff
@@ -230,6 +230,11 @@ std::vector<IdTable> convertToVector(
   return result;
 }
 
+// match the contents of a `vector<IdTable>` to the given `tables`.
+auto matchIdTables(ad_utility::SimilarTo<IdTable> auto const&... tables) {
```

Comment:
Same thing here: If this gets renamed to `matchesIdTables`, you can read the test as "expect that x matches id tables y..."